### ISSUE TITLE
Wont load under Cura 15.02.1 on windows 8.1

### ISSUE THREAD

**Issue by Razzle69** [scotthraban/octoprint-printer-connection-plugin-for-cura#5]
I followed your install instructions but the plugin wont load in Cura 15.02.1 running on 64bit windows 8.  All the other plugins load.  But there is no sign of the printerconnection plugin when Cura is started.

**Comment by scotthraban** [comment]
Did you apply the two pull requests that I list as dependencies, and are you running a devel branch of OctoPrint (i.e. one with the new'ish API)?

Be aware that the plugin does not show up like the other plugins, since the only currently supported plugins are for changing the gcode that is output.

You should expect to see that when you have items loaded on the platter, that the save icon changes to what looks more like a wifi icon, which will send the gcode to the OctoPrint when clicked.

Can you send me the Cura log?

**Comment by Razzle69** [comment]
I am running a devel version of octoprint.  I missed the cura dependencies.  Have to figure out how to do a pull request into cura on windows.  I used the package install for cura.  So is the octoprint_configuration.json file part of the cura pull?  It is not in the printerconnection package.

**Comment by scotthraban** [comment]
I don't believe that you can do a pull request straight into an installation, I am pretty sure you would have to either create your own fork, and pull in my pull requests, then build your own distribution of Cura to install, or manually change the files in your installation to add the changes in the pull request.

Neither of these are very straightforward :(

@Nandox7 @Gravity @quillford @fieldOfView @Razzle69 Can I request that you comment on the pull requests that you would like to see these changes in Cura so we can see if we can get them included into Cura for real? The pull requests seem to have stalled, I have responded to all the changes that were asked for, but have not heard anything for a while from the Cura developer.

The json file is auto created the first time that Cura is started with all the changes needed to start the plugin.

**Comment by Nandox7** [comment]
You can checkout the cura code and later cherry-pick the pull requests. That is what I did to try it out.

@scotthraban funny that you mention  that as I was thinking in doing that yesterday. I don't understand why don't they go forward.
